### PR TITLE
Create/Edit MT Modals

### PR DESCRIPTION
**Pull request by hang-m-cao**

### CODE REVIEW

File: routes/teachers.js [hunk 1]
```diff
@@ -59,36 +59,38 @@ router.get('/site/:siteId', async (req, res) => {
...
+    const newUser = await firebaseAdmin.auth().createUser({
+      email,
+      emailVerified: true,
+      password,
+    });
 
     const newTeacher = await pool.query(
       `INSERT INTO tlp_user
         (firebase_id, first_name, last_name,
-        phone_number, email, position, active, notes)
```
Comment (anikait-rao) [inline routes/teachers.js:73]: Why did you remove phone number? It is still an optional field right?

Comment (anikait-rao) [reply routes/teachers.js:73]: Also add an IF check to the PUT request to handle the phone number potentially being null

**Review by anikait-rao — CHANGES_REQUESTED** [review]
see comments

File: routes/teachers.js (hunk 1 above)
Comment (hang-m-cao) [reply routes/teachers.js:73]: There's no phone number in the [modal](https://www.figma.com/file/O2igPzMS1fZPzjmOxlo6Mc/TLP-Wireframes?node-id=4334%3A118134) for teacher account creation as mentioned [here in slack](https://ctcuci.slack.com/archives/C02JMK343V4/p1652903148531539?thread_ts=1652771107.951369&cid=C02JMK343V4)

**Review by anikait-rao — APPROVED** [review]
seems right
